core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -1320,6 +1331,175 @@ protected SqlNode performUnconditionalRewrites(
     return node;
   }
 
+  private SqlNode performHiveUdtfRewrite(SqlNode node) {
+    Map<SqlSelect, UdtfInfo> selectUdtfInfo = new HashMap<>();
+     SqlNode rewirte = performHiveUdtfRewriteInternal(node, selectUdtfInfo);
+   //  System.out.println("before:\n" + node.toString() +" \nafter:\n" + rewirte.toString());
+    return node;
```

Comment:
Should remove this useless comment.

Reply:
Sorry to push this test code.remove now!